src/constants.js

```diff
@@ -23,3 +23,14 @@ export const NOTIFICATION_MESSAGES = {
   duplicating: 'Duplicating',
   deleting: 'Deleting',
 };
+
+const COURSE_CREATOR_STATUSES = {
```

Comment:
In the constants above, the export is done before the declaration of the constants. Let's follow this rule